themes/app/src/scss/components/_alert.scss

```diff
@@ -0,0 +1,13 @@
+$gap: 20px;
+
+.alert-box {
+    background-color: gray;
```

Comment:
Could use a SCSS variable here

Reply:
I leave it there, as it will be changed during style setup.